libmozdata/hgmozilla.py

```diff
@@ -241,3 +246,41 @@ def get(paths, channel='nightly', node='tip'):
             FileInfo(queries=queries).wait()
 
         return data
+
+
+class HGMozilla(object):
```

Comment:
Right now it's similar to HGFileInfo since we just have get_filelog, but we could add other stuff, so I'd prefer to keep it separately.